### DISCUSSION TITLE
Error solving the challenge using a Raspberry Pi

### DISCUSSION THREAD

**Discussion by soflyj** [FlareSolverr/FlareSolverr#1311] [Q&A]
Please can I get assistance before this post is blocked or removed or I am banned.

[Jackett cannot resolve endpoint, http://192.168.x.x:8191/v1 500 Internal Server Error?](https://github.com/FlareSolverr/FlareSolverr/issues/1305)

I have read through the mentioned duplicated issues (and several other issues), tried everything posted. And I've even learnt the basics just to debug Python.
There are finer details are on the issue, but I'll some it up.

I am using a Raspberry Pi 4, running Raspberry OS Lite. Running Jackett and Prowlarr.

`{"msg": "FlareSolverr is ready!", "version": "3.3.21", "userAgent": "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/120.0.0.0 Safari/537.36"}`

`2024-07-31 20:07:59 INFO     Incoming request => POST /v1 body: {'maxTimeout': 55000, 'cmd': 'request.get', 'url': 'https://ext.to/latest/'}
2024-07-31 20:08:05 INFO     Challenge detected. Title found: Just a moment...
2024-07-31 20:08:55 ERROR    Error: Error solving the challenge. Timeout after 55.0 seconds.
2024-07-31 20:08:55 INFO     Response in 56.515 s
2024-07-31 20:08:55 INFO     192.168.1.10 POST http://192.168.1.10:8191/v1 500 Internal Server Error`

**Comment by ilike2burnthing** [comment]
> ### Is there already an issue for your problem?
> - [X] I have checked older issues, open and closed

I suggest you try again. See the pinned issues.
